code/modules/clothing/neck/_neck.dm

```diff
@@ -177,16 +177,55 @@
 
 /obj/item/clothing/neck/petcollar
 	name = "pet collar"
-	desc = "It's for pets. Though you probably could wear it yourself, you'd doubtless be the subject of ridicule."
+	desc = "It's for pets. Though you probably could wear it yourself, you'd doubtless be the subject of ridicule. It seems to be made out of a polychromic material."
 	icon_state = "petcollar"
+	alternate_worn_icon = 'icons/mob/neck.dmi'
```

Comment:
alternate_worn_icon is only necessary if the object has a non-standard icon path. except in edge cases like this where the item's procs and other code rely on having that icon. add a comment denoting such to prevent confusion from other contributors and maintainers alike

Reply:
From the polychromic clothing code. https://github.com/Citadel-Station-13/Citadel-Station-13/blob/fcc8212b728885674152b43a880cb97e0a132ebb/modular_citadel/code/modules/clothing/under/polychromic_clothes.dm#L33
`/obj/item/clothing/under/polychromic/worn_overlays(isinhands, icon_file) //this is where the main magic happens. Also mandates that ALL polychromic stuff MUST USE alternate_worn_icon`